language/English/strings.po

```diff
@@ -8838,7 +8850,43 @@ msgctxt "#19289"
 msgid "Hide group"
 msgstr ""
 
-#empty strings from id 19290 to 19498
+#. Used on recordings context menu
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19290"
+msgid "Undelete"
+msgstr ""
+
+#. Used on recordings context menu
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19291"
+msgid "Delete permanently"
+msgstr ""
+
+#. Used on recordings context menu
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19292"
+msgid "Delete all permanently"
+msgstr ""
+
+#. Used on yes no dialog
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19293"
+msgid "Delete all recordings permanently?"
+msgstr ""
+
+#. Used on yes no dialog header
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19294"
+msgid "Confirm undelete"
```

Comment:
Can be removed, is a bit to much ;-). Thought on beginto make sure thats right before on addon some work becomes performed.